java-extensions/hadoop-ext/pom.xml

```diff
@@ -44,6 +44,24 @@
             <version>${iceberg.version}</version>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/com.google.cloud.bigdataoss/gcs-connector -->
+        <dependency>
```

Comment:
no need to place it here. 

put it in common-lib. hadoop-ext should be as simple as possible.